packages/design-tokens/docs/pages/how-to-switch-themes.stories.mdx

```diff
@@ -0,0 +1,20 @@
+import { Meta } from '@storybook/addon-docs/blocks'
+import { LinkToStory, CodeBlock } from '../DocsComponents'
+import switchingThemesExample from '!!raw-loader!./examples/switchingThemes.tsx'
+import gettingStartedPage from './how-to-setup.stories.mdx'
+
+<Meta title="How-to guides/How to switch themes" parameters={{ docsOnly: true }} />
+
+
+# How to switch themes
+
+## 1. Setup a ThemeManager
+
+Ensure you have followed the steps in <LinkToStory storyModule={gettingStartedPage} /> to setup your application for using design tokens.
+
+## 2. Use `setAndApplyTheme`
+
+You can use the method `setAndApplyTheme` to change the current theme, to either a preset like Zen or Heart, or a customised object.
```

Comment:
```suggestion
You can use the `setAndApplyTheme` method to change the current theme, to either a preset like Zen or Heart, or a customised object.
```

Reply:
I'm also pondering if showing "custom" is worth it in this how to. Thinking about [this](https://documentation.divio.com/how-to-guides/#leave-things-out):

> Leave things out
> They don’t need to mention everything that there is to mention either, just because it is related to the topic. A bloated how-to guide doesn’t help the user get speedily to their solution.